brainscore_core/plugin_management/conda_score.py

```diff
@@ -39,29 +39,29 @@ def score_in_env(self) -> 'subprocess.CompletedProcess[bytes]':
         return completed_process
 
     @staticmethod
-    def read_score(library_path: Path):
-        score_path = CondaScore._score_path(library_path)
+    def read_score(library_path: Path, env_name: str):
```

Comment:
```suggestion
    def read_score(library_path: Path, env_name: str) --> Score:
```
Just seeing the missing return value now; I think it makes sense to add with this PR

Reply:
@benlipkin I messed up the arrow in my initial comment and put `-->` instead of `->`, so there's probably a syntax error in the file now